src/libs/antares/utils.cpp

```diff
@@ -68,4 +72,41 @@ void BeautifyName(YString& out, AnyString oldname)
     out.trim(" \t\r\n");
 }
 
+template<>
+void TransformNameIntoID(const AnyString& name, std::string& out)
+{
+    Yuni::String yuniOut;
+    TransformNameIntoID(name, yuniOut);
+    out = yuniOut;
+}
+
+void BeautifyName(std::string& out, const std::string& oldname)
+{
+    YString yuniOut;
+    BeautifyName(yuniOut, oldname);
+    out = yuniOut.c_str();
+}
+
+char charToLower(char in)
```

Comment:
I may be wrong but it seems that this function should be in the anon namespace

Reply:
Indeed: it has now been removed in the "boost version" anyway.